src/main/scala/mesosphere/marathon/core/task/tracker/impl/steps/UpdateTaskTrackerStepImpl.scala

```diff
@@ -1,65 +1,37 @@
 package mesosphere.marathon.core.task.tracker.impl.steps
 
 import com.google.inject.Inject
-import mesosphere.marathon.MarathonSchedulerDriverHolder
 import mesosphere.marathon.Protos.MarathonTask
 import mesosphere.marathon.core.task.tracker.TaskStatusUpdateStep
-import mesosphere.marathon.state.{ AppDefinition, PathId, Timestamp }
+import mesosphere.marathon.state.{ PathId, Timestamp }
 import mesosphere.marathon.tasks.TaskTracker
 import org.apache.mesos.Protos.TaskStatus
-import org.slf4j.LoggerFactory
 
 import scala.concurrent.Future
-import scala.util.control.NonFatal
 
 /**
   * Update task tracker corresponding to the event.
   */
 class UpdateTaskTrackerStepImpl @Inject() (
-    taskTracker: TaskTracker,
-    marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder) extends TaskStatusUpdateStep {
-
-  private[this] val log = LoggerFactory.getLogger(getClass)
+    taskTracker: TaskTracker) extends TaskStatusUpdateStep {
 
   def name: String = "updateTaskTracker"
 
-  //TODO: fix style issue and enable this scalastyle check
-  //scalastyle:off cyclomatic.complexity method.length
   def processUpdate(
```

Comment:
Removed the cyclomatic complexity! +1
And I have to admit scalastyle is right - it reads a lot easier...